notebook/tests/selenium/quick_selenium.py

```diff
@@ -0,0 +1,43 @@
+from selenium.webdriver import Firefox
```

Comment:
Let's have a module level docstring to explain that these utilities are not used by the tests, but they're intended for interactive exploration while writing tests.

Reply:
I had intended to add this.